bundle/src/bootstraps/standalone.commercial.ts

```diff
@@ -231,6 +237,12 @@ const chooseAdvertisingTag = async () => {
 	}
 };
 
+if (isInVariantSynchronous(elementsManager, 'variant')) {
+	void import('../projects/commercial/modules/elementsManager').then(
```

Comment:
Up to you but you could add a comment to instruct webpack to give this chunk a name e.g.

```
void import(/* webpackChunkName: "elements-manager" */ ...)
```